README.md

```diff
@@ -86,7 +86,7 @@ rake r10k:syntax
 rake r10k:validate                                                              # Validate the git urls and branches, refs, or tags
 rake release                                                                    # Release via GitHub Actions
 rake release:prepare                                                            # Prepare a release
-rake release_checks                                                             # Runs all necessary checks on a module in preparation for a release
+rake release_checks                                                             # Runs all necessary checks on a module in preparation for a r...
```

Comment:
those changes look odd. Did you run `rake -T` with a too small terminal?

Reply:
oh, i did... gonna fix that